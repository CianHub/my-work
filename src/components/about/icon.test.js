// about.icon.test.js
import React from "react";
import Icon from "./icon";
import renderer from "react-test-renderer";
import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";

test("Icon is rendered", () => {
  const component = renderer.create(
    <Icon icon="users" title="Spaceship" text="This goes to space!" />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
