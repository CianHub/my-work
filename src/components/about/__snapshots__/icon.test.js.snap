// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon is rendered 1`] = `
<div
  className="service-box mt-5 mx-auto"
>
  <h3
    className="mb-3"
  >
    Spaceship
  </h3>
  <p
    className="text-muted mb-0"
  >
    This goes to space!
  </p>
</div>
`;
